engine/src/main/java/com/datatorrent/stram/client/StramAgent.java

```diff
@@ -421,7 +421,11 @@ private StramWebServicesInfo retrieveWebServicesInfo(String appId)
         permissionsInfo = new JSONObject(IOUtils.toString(is));
       } catch (IOException ex) {
         // ignore if file is not found
-        LOG.info("Exception in accessing permissions.json", ex);
+        if (LOG.isDebugEnabled()) {
```

Comment:
@tweise My understanding is that it is very rear that permission.json file will be present, in this case, will it be better to have a check?